### ISSUE TITLE
Clound Foundry: service binding for [elasticsearch] is missing property: [port]

### ISSUE THREAD

**Issue by schefDev** [swisscom/backman#83]
Honestly, I don't know if it's a bug in the Cloud Foundry service I'm using, but I have this exact case here.
```
   2023-12-21T13:09:19.25+0100 [APP/PROC/WEB/0] OUT level=error msg="service binding for [elasticsearch] is missing property: [port]"
   2023-12-21T13:09:19.25+0100 [APP/PROC/WEB/0] OUT level=error msg="service binding for [elasticsearch] is not valid"
``` 

Cause, the host (in credentials of the service binding) has no port:
```
"host": "https://xxx.elasticsearch.com",
``` 
instead of 
```
"host": "https://xxx.elasticsearch.com:443",
``` 

To fix that in the backman if it finds approval, the well-known port in enrichBinding could be set according to the schema/protocol. 
If I find the time, I will submit a pull request.

Add https://github.com/swisscom/backman/blob/develop/service/service.go#L139
```
// set well-known port based on scheme if still missing
if binding.Port == 0 {
	if strings.EqualFold(u.Scheme, "https") {
		binding.Port = 443
	}
	if strings.EqualFold(u.Scheme, "http") {
		binding.Port = 80
	}
}
```
Workaround: Downgrade to backman version 1.32.2

BR Christian

**Comment by schefDev** [comment]
I just noticed that this case (no port in host) is common with older elasticsearch service instances.

**Comment by schefDev** [comment]
Hi @JamesClonk or other contributors, 

if you ever have time, it would be cool if you could take a look at this trifle. 

Thanks and BR,
Christian

PS: Just so it's not lost if no one is notified. 👻

**Comment by JamesClonk** [comment]
@schefDev I'm on it, thanks for the ping! :+1: 
(Indeed, I don't seem to get notifications of issues or PRs unfortunately :open_mouth:)

**Comment by schefDev** [comment]
Thanks @JamesClonk! 👍 

Fixed in Release [v2.1.1](https://github.com/swisscom/backman/releases/tag/v2.1.1)
